packages/create-block/lib/prompts.js

```diff
@@ -1,7 +1,17 @@
 /**
  * External dependencies
  */
-const { isEmpty, upperFirst } = require( 'lodash' );
+const { isEmpty } = require( 'lodash' );
+
+/**
+ * Capitalizes the first letter in a string.
+ *
+ * @param {string} str The string whose first letter the function will capitalize.
+ *
+ * @return {string} Capitalized string.
+ */
+const upperFirst = ( [ firstLetter, ...rest ] ) =>
+	firstLetter.toUpperCase() + rest.join( '' );
```

Comment:
Should we handle empty strings etc... that will break the `firstLetter.toUpperCase()`?

Reply:
I think that's not necessary for 2 reasons:

1. There are default values if empty string is provided:

https://github.com/WordPress/gutenberg/blob/remove/lodash-upper-first/packages/create-block/lib/templates.js#L38

2. `upperFirst()` will be called only on non-falsy values, so it won't be invoked on empty strings:

https://github.com/WordPress/gutenberg/blob/remove/lodash-upper-first/packages/create-block/lib/prompts.js#L50

WDYT?